opal_server/server.py

```diff
@@ -151,6 +151,14 @@ def _init_fast_api_app(self):
         """
         inits the fastapi app object
         """
+        if "ENABLE_MONITORING" in os.environ and os.environ["ENABLE_MONITORING"].lower() != "false":
+            from ddtrace import patch, config
+            # Datadog APM
+            patch(fastapi=True)
+            # Override service name
+            config.fastapi["service_name"] = "opal-client"
+            config.fastapi["request_span_name"] = "opal-client"
+
```

Comment:
same comments as in the sidecar diff:
- use Confi to define the config var
- put this in a separate method - self._configure_tracing()